pkg/rkeworker/docker.go

```diff
@@ -280,6 +286,10 @@ func natPortSetToSlice(args map[nat.Port]struct{}) []nat.Port {
 }
 
 func runLogLinker(ctx context.Context, c *client.Client, containerName string, p v3.Process) error {
+	if ignoreWindows(ctx) {
```

Comment:
I feel like this can be detected using runtime.GOOS.

Reply:
Ok, got